db/migrate/20220222114752_create_orders.rb

```diff
@@ -2,7 +2,7 @@ class CreateOrders < ActiveRecord::Migration[6.1]
   def change
     create_table :orders do |t|
       t.references :employee, null: false, foreign_key: true
-      t.references :reward, null: false, foreign_key: true
+      t.references :reward, foreign_key: true
```

Comment:
I had to correct this to remove the reward despite the order.
I mean here that a certain reward is no longer available, can be deleted, but is still visible in the order history.